### ISSUE TITLE
Accessibility: Voiceover under MacOS doesn't read block suggestions as of Sonoma

### ISSUE THREAD

**Issue by ashleycox** [WordPress/gutenberg#56618]
### Description

When inserting an empty block by pressing enter in the editor, and then pressing the forward slash to bring up a list of block suggestions, the VoiceOver screenreader on MacOS no-longer reads the list of suggestions. This issue started occurring with macOS Sonoma, it was working fine in Ventura. Tested with both Safari and Chrome with identical behaviour, and on 2 different machines, both after resetting VoiceOver preferences to their defaults. Not sure what's causing this?

### Step-by-step reproduction instructions

Enable Voiceover. load a new page / post in a browser. create an empty block and press /. try to read the list of blocks with the arrow keys as instructed by Voiceover when / is pressed.

### Environment info

Wordpress Version 6.4.1, macOS Sonoma 14.1.1, Safari Version 17.1 (19616.2.9.11.7).

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by talldan** [comment]
Thanks for reporting this. It seems a little bit intermittent when I tested, it worked sometimes, and other times it didn't. (just in case anyone else is testing and can't reproduce, keep trying)

I also noticed that sometimes the caret jumps back to the post title after inserting a first block, but that's also a little inconsistent and only seemed to happen with voiceover and safari.

**Comment by ashleycox** [comment]
I've been looking into this some more. In Chrome on a windows PC using NVDA, it works perfectly. I downloaded the latest stable version of Firefox for the Mac, and it works in Firefox too. Perhaps there is a difference in the way the keyboard (in particular the arrow keys) are handled in Firefox vs Chrome and Safari on the Mac, or a difference in teh display of the block editor interface?
